experiments/AMIP/modular/components/atmosphere/climaatmos_init.jl

```diff
@@ -109,16 +158,11 @@ function coupler_surface_setup(
 end
 
 """
-    compute_atmos_turbulent_fluxes!(atmos_sim::ClimaAtmosSimulation, csf)
+    get_new_p(atmos_sim::ClimaAtmosSimulation, csf)
 
-Computes turbulent surface fluxes using ClimaAtmos's `update_surface_conditions!`. This
-requires that we define a new temporary parameter Tuple, `new_p`, and save the new surface state
-in it. We do not want `new_p` to live in the atmospheric model permanently, because that would also
-trigger flux calculation during Atmos `step!`. We only want to trigger this once per coupling
-timestep from ClimaCoupler.
+Returns a new `p` with the updated surface conditions.
 """
-
-function compute_atmos_turbulent_fluxes!(atmos_sim::ClimaAtmosSimulation, csf)
+function get_new_p(atmos_sim::ClimaAtmosSimulation, csf)
 
     function modified_atmos_p(atmos_sim, csf_sfc)
```

Comment:
not sure about the tab here

Reply:
I think this is a function defined within `get_new_p`, so the tab is okay

Reply:
Yeah, I saw that later. I thought GH was tricking me :) 
But we should probably avoid the use of encapsulated functions. Is it really necessary here? Or we can define this inner function outside?

Reply:
It seems like nested functions used to be slower in Julia some time ago but that seems not to be a concern anymore. So if this is not aggravating perf/allocations, we can leave it as nested, if it makes the arg passing easier.

Reply:
There's no real reason anymore to keep it within the local scope, so happy to move it out!